tempsensor/tempDB.py

```diff
@@ -0,0 +1,295 @@
+import abc
+import sqlite3
+import logging
+import argparse
+import os
+import thingspeakinfo as c
+from datetime import datetime
+
+FIRST_ROW = 0
+SINGLE_RECORD = 1
+
+class SqliteDB(metaclass=abc.ABCMeta):
+	"""
+	DB to store data
+	"""
+
+	def __init__(self, db_file, name):
+		"""
+		Initializes TempDB Context Manager
```

Comment:
I think the comment here needs to be updated to the general name; however, I was thinking that maybe I could separate the SqliteDB into its own file in a common directory (maybe this weekend :) )